### ISSUE TITLE
download

### ISSUE THREAD

**Issue by moranhq** [SaherBlueEagle/BlueEagle-XPR-Open-Source#2]
Please how can i download the packed jar also anyway i can contact you?

thank you

**Comment by SaherBlueEagle** [comment]
hello bro , you mean source code for jar trojan , it is pretty easy , the tutorials / blogger also may help you enough bro , and if it`s not , you can tell me here again about the inquiry in details , good luck bro , don`t forget to take alook with Endless tool , also the r*n*o*war* and trojan apk builders are isolated from the main jar but inside the repository there !! , Best Regards ;)

**Comment by moranhq** [comment]
yes i mean the zip or standalone exe where i can build the trojan itself and also see when the trojan connects

**Comment by SaherBlueEagle** [comment]
better download the release (standlone exe) the source here is for developers . because it need some requirements known by them which are  must to make the source code compiled into release and rum smoothly ;)

**Comment by moranhq** [comment]
yes i can not find the release and i am not a developer to compile

can you compile for me and send me me download link?

**Comment by SaherBlueEagle** [comment]
okay bro , dear , 
download of release is here since years 
https://github.com/SaherBlueEagle/XPR-2020-Free
and its available even before source was released bro

**Comment by moranhq** [comment]
thanks mate i am grateful  if any way i can contact you please let me know thanks
i want to discuss something with you
